### PR TITLE
Remove TXINGs check and modified to run under RH8

### PR DESCRIPTION
**Pull request by Gregg140**
lr and all associated Load Review programs were modified to run under
RH8.

lr was  modified to eliminate the check for the TXINGS commands
as part of the Perigee Passage ECS measurements.

### CODE REVIEW

**Comment by cegrant** [comment]
I thought we were deleting the copy to luke-v from history-files altogether?
Line 197 of history-files points to a specific version of ACE-update, rather than the linked one.  Is that intentional?
Otherwise, I think this looks OK.

**Review by jazan12 — APPROVED** [review]
I had one small comment for you to double-check in there and noticed as I see Catherine had pointed out already the question of whether or not we are copying to luke-v or not.  Otherwise, looks good.  I'm clicking approve under assumption that either both those are as intended or the trivial fix will be done.

**Comment by Gregg140** [comment]
Good catch on the hard coded call to ACE-update when running in test mode. It's in the productin version also. I'll remove it.

Removing the copies to luke-v wasn't done because I was too far along in testing to add it when we decided to eliominate the copy. I can add it now since I have to re-test everything.

File: history-files.pl [hunk 1]
```diff
@@ -822,8 +973,8 @@ ()
       {  # Test is part of regression suite
 	$test_dir = "/pool14/duderg/src/regress/history-files";
 	$ops_dir="$base_dir/acisdude/reg_script/${choice}test";
-	
-	$ACE_UPDATE="${test_dir}/ACE-update.pl -test $ops_dir";
+
+	$ACE_UPDATE="${test_dir}/ACE-update-v2.15.pl -test $ops_dir";
```
[the remark this answers is not in the document]
Comment (Gregg140) [reply history-files.pl]: Fixed

**Comment by Gregg140** [comment]
Please re-review.

Additional changes were made: 

1) LRHS - updated to Python3
2) Removed the call to fptemp_sens_finder.pl
3) Print a line in the load review log indicating what thermal model is about to be run
4) History files were no longer copied  to luke-v and
                         han-v
lr and the associated Load Review programs, shown below, were modified, as
necessary, and tested,  to run under RH8 and Python3.

In addition, the following updates were made:

lr -  eliminated the check for the TXINGS commands as part of the
      Perigee Passage ECS measurements.
   - Removed code that called fptemp_sens_finder.pl; function no
     longer needed

LRHS - Commented statements removed.

run_models.pl - removed the call to make_dhheater_history.csh as it's
                        no longer needed.
		      - Print a line indicating what thermal model is about to be run

acis-backstop.pl - Perigee Passage TXING Quiet checks removed

history-files.pl - history files were no longer copied  to luke-v and
                         han-v

ACE-update.pl - history files were no longer copied  to luke-v and
                         han-v

File: ACE-update.pl [hunk 2]
```diff
@@ -0,0 +1,549 @@
...
+    $isBackup = 1;
+}
+else{
+    $isBackup = 0;
+}
+$base_dir = </data/acis${appx}>;
+
+# FLU-MON directory definitions
+
+$han_flu=GetFluMon('han-v');
```
Comment (jazan12) [inline ACE-update.pl:86]: Do you still want this defined?

File: ACE-update.pl [hunk 3]
```diff
@@ -0,0 +1,549 @@
...
+
+while (($key, $value) = each(%history_files)){
+    $hist_fls_str .= "$dir/${key}.dat ";
+   #----------------------------------------
+    #Change the group to acisops
+    #----------------------------------------
+    &chgrp_by_name("acisops","$dir/$key.dat");
+}
+
+$HEAD_flu = "han-v:$xcanuck_flu"; # Global history file directory
```
Comment (jazan12) [inline ACE-update.pl:202]: Looks like you're using han-v for the global; is that still how you want to do it, or should han-v be excised altogether?

File: history-files.pl [hunk 4]
```diff
@@ -102,13 +127,16 @@
...
-$han_v_flu=GetFluMon('han-v');
+
+$aciscdp_v_flu=GetFluMon('aciscdp-v');
+
+$acisway_v_flu=GetFluMon('acisway');
+$ishmael_v_flu=GetFluMon('ishmael');
 
 $colossus_v_flu=GetFluMon('colossus-v');
-$acisocc_v_flu=GetFluMon('acisocc-v');
+#$han_v_flu=GetFluMon('han-v');
```
Comment (jazan12) [inline history-files.pl:137]: Do you want these definitions here for luke-v and han-v?  I don't think they're needed now, right?

**Review by jazan12 — APPROVED** [review]
I see some definitions that look extraneous that I think could be deleted but nothing that looks like it would be problematic to run.

**Review by cegrant — APPROVED** [review]

File: ACE-update.pl (hunk 2 above)
Comment (Gregg140) [reply ACE-update.pl:86]: Commented out. Might want han-v to be defined in the future.

File: history-files.pl (hunk 4 above)
Comment (Gregg140) [reply history-files.pl:137]: Note that it is commented out. Keeping the line in there to get flumon file targets in case it's needed in the future.

**Pull request merged**